HydraIRC/include/HydraControls/HydraSplitter.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 template <bool t_bVertical = true>
-class CHydraSplitterWindowT : public CSplitterWindowImpl<CHydraSplitterWindowT<t_bVertical>, t_bVertical>
+class CHydraSplitterWindowT : public CSplitterWindowT<t_bVertical>
```

Comment:
this change seems somehow unrelated to this library upgrade.  can you explain it or remove the change?

Reply:
WTL 9.0+ Made changes to splitter windows, specifically this one from the 9.0 release notes:

"Splitter Window: Changed orientation from template argument to data member to reduce memory use"

Pre-change with WTL9.0+ instead of WTL8 included, results in "error C2923: 'WTL::CSplitterWindowImpl' : 't_bVertical' is not a valid template type argument for parameter 'TBase'"

Reply:
Added code comment too clarifying - it must have gotten lost while I was sorting out the whole branching model in my head.